### DISCUSSION TITLE
A name for our plugin - currently plugin-convention

### DISCUSSION THREAD

**Discussion by bigopon** [aurelia/aurelia#1404] [Ideas]
At the moment, we have a package `@aurelia/plugin-convention`. It compiles & glues a pair of view and view models together via some extra generated code. Because of this nature, adding Hot Module Reload (hmr) feature to Aurelia feels like a part of this package, at least at the start. Though as the result of adding this feature, `plugin-convention` is no longer an appropriate name. We need to figure out some better name for it.

Some early, obvious name suggestions:
- `processor`
- `pre-processor`

Related PR #1400

**Comment by jwx** [comment]
Might `plugin-build` be an option?

**Comment by Sayan751** [comment]
As HMR cannot really be considered as a `pre-processor` I would suggest avoiding that. `processor` or `build` seems quite alright. `plugin-build` seems to give a hint that this useful for building plugins. The earlier name `plugin-convention` seems to carry the same connotation.
